terraform/hub/deploy.sh

```diff
@@ -10,4 +10,7 @@ ROOTDIR="$(cd ${SCRIPTDIR}/../..; pwd )"
 terraform -chdir=$SCRIPTDIR init --upgrade
 
 echo "Deploy Hub cluster"
+terraform -chdir=$SCRIPTDIR apply -target="module.eks" -auto-approve
+terraform -chdir=$SCRIPTDIR apply -target="module.eks_blueprints_addons" -auto-approve
+terraform -chdir=$SCRIPTDIR apply -target="module.gitops_bridge_bootstrap" -auto-approve
```

Comment:
what's the argocd permission error that we need to do this now?
Can we put the dependson inside the terraform to force the ordering?